packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -91,5 +91,6 @@ export const handler = async ({ force }) => {
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```